File: backend/src/main/java/com/team01/backend/domain/user/dto/LoginRequest.java
package com.team01.backend.domain.user.dto;

import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.Size;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;


@Getter
@NoArgsConstructor
@AllArgsConstructor
public class LoginRequest {

    @NotBlank(message = "이메일은 필수 입력 값입니다.")
    @Email(message = "올바른 이메일 형식이 아닙니다.")
    private String email;

    @NotBlank(message = "비밀번호는 필수 입력 값입니다.")
    @Size(min = 4, message = "비밀번호는 최소 4글자 이상이어야 합니다.") // [과제] 로그인 시에도 길이 체크
    private String password;
}

File: backend/src/main/java/com/team01/backend/domain/user/dto/LoginRequest.kt
package com.team01.backend.domain.user.dto

import jakarta.validation.constraints.Email
import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.Size

data class LoginRequest(
    @field:NotBlank @field:Email val email: String,
    @field:NotBlank @field:Size(min = 4) val password: String
)
